l2-contracts/contracts/common/libraries/DataEncoding.sol

```diff
@@ -67,20 +67,20 @@ library DataEncoding {
     /// @param _sender The asset deployment tracker address.
     /// @return The encoded asset data.
     function encodeAssetId(address _tokenAaddress, address _sender) internal view returns (bytes32) {
-        return keccak256(abi.encode(block.chainid, _sender, _tokenAaddress));
+        return keccak256(abi.encode(L1_CHAIN_ID, _sender, _tokenAaddress));
```

Comment:
the same `DataEncoding` library has two methods named `encodeAssetId` with different params and very different logic, where in one of those  `L1_CHAIN_ID` is used and in the other `block.chainid` is used. They should have semantically same logic

Secondly, it seems like we hardcode  `L1_CHAIN_ID = 1` here which wont work on local network/testnets